src/net_processing.cpp

```diff
@@ -3019,6 +3019,15 @@ void PeerManager::ProcessMessage(CNode& pfrom, const std::string& msg_type, CDat
             if (!fRejectedParents) {
                 const auto current_time = GetTime<std::chrono::microseconds>();
 
+                // To save bandwidth during IBD, stop orphan processing early. With the state of
+                // our UTXO set far from tip, we might not have yet the unspent output spend by this
+                // transaction or this transaction is already double-spend but not yet seen block.
+                // Further, we can't assert accuracy of its feerate.
+                //
+                // Orphan transaction is discarded. Note, we don't both to add id to recentRejects
+                // as it's expected to be quickly clean up during IBD.
+                if (ChainstateActive().IsInitialBlockDownload()) return;
```

Comment:
Does the scope resolution operator need to be used here like everywhere else? `if (::ChainstateActive().IsInitialBlockDownload()) return;`

Reply:
Good point! I'll update once PR got Concept ACK/NACK.